src/nat/agent/router_agent/register.py

```
@@ -85,7 +79,7 @@ async def _response_fn(input_message: str) -> str:
 
         except Exception as ex:
             logger.exception("%s Router Agent failed with exception: %s", AGENT_LOG_PREFIX, ex)
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)